packages/@expressive-code/plugin-shiki/test/rendering.test.ts

```diff
@@ -66,4 +80,25 @@ describe('Renders syntax highlighting', () => {
 		},
 		{ timeout: 5 * 1000 }
 	)
+
+	test(
+		'Supports ansi',
+		async ({ meta: { name: testName } }) => {
+			await renderAndOutputHtmlSnapshot({
+				testName,
+				testBaseDir: __dirname,
+				fixtures: buildThemeFixtures(themes, {
```

Comment:
Overkill? Keep it to one theme so we can validate colors as well?

Reply:
I think rendering multiple themes is perfectly fine! I always like checking the generated snapshots to see how the various color themes affect the rendered output.

However, I think we should check the rendered html for a few expected colors in at least one theme. You could do this by also destructuring the `theme` property in the `blockValidationFn` arguments, check for a particular theme name and then expect the html to contain the desired color. To ensure this check is actually hit (the theme name did actually match and get rendered), you could set some test-scoped variable and check if it has the expected value at the end of the test case.